client/Dockerfile

```diff
@@ -35,6 +35,6 @@ RUN mkdir -p /dex/.dexc && chown 1000 /dex/.dexc
 USER 1000
 COPY --from=gobuilder /root/dex/client/cmd/dexc/dexc ./
 COPY --from=gobuilder /root/dex/client/cmd/dexcctl/dexcctl ./
-COPY --from=gobuilder /root/dex/client/webserver/site ./site
+COPY --from=gobuilder /root/dex/client/webserver/site/dist ./site/dist
```

Comment:
The subfolders of "site" that are used are: dist, src/font, src/html, src/img.

However, since by default these resources are all embedded into the dexc binary, I think you can actually remove this entire webserver/site line.

Once in the past the pkg.sh script did this: https://github.com/decred/dcrdex/blob/83ccb74ff404e7029d2d3f64c1b57cdbae64c7f5/pkg.sh#L45-L46
But now it's enough to make sure to do the npm build steps *before* building dexc: https://github.com/decred/dcrdex/blob/8d99975b34e4ed372953ee39f5fbc659b2302918/pkg.sh#L16-L17

Reply:
You could still grab these folder though if for any reason you want to run dexc with he `--no-embed-site` option that has it search on-disk, but I imagine that's not something you'd be doing with the Dockerized distro.

Reply:
Thanks for clarifying that - the `site` content isn't needed at all then.

Reply:
BTW this was needed as Umbrel aims to run the smallest possible images (preferably without any source material), and this is good practice anyways.